amida-tech/mcp-data-client
# Mcp 181 jf client cleanup

### Related tickets:
MCP-179, MCP-180, MCP-181.

### What's changing? Any breaking changes?
Nothing. Added tests for the data service, changed function names per Grant's request, added a working `Dockerfile`. 

### Manual test cases?
Run `yarn test` and confirm it's all working. Then run `docker build -t mcp-data-client .` followed by `docker run -p 3000:80 -e REACT_APP_MCP_DATA_MAPPING_UTIL=http://localhost:5000/ mcp-data-client` to run a docker container of the front end. If you set your port for the backend to something else, obviously adjust the port accordingly. 

### Any additional context/background?
-

### Screenshots (if appropriate):
-
